src/main/docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM frolvlad/alpine-oraclejdk8:slim
+VOLUME /tmp
+ADD test-env-runner-0.1.0.jar app.jar
```

Comment:
version numbers will be changed continuously. Gradle plugin could generate Dockerfile dynamically